src/components/MaxCTA/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Link from 'components/Link'
+
+type MaxCTAProps = {
+    className?: string
+    children?: React.ReactNode
+    question: string
+}
+
+export const MaxCTA = ({ className = '', children, question }: MaxCTAProps): JSX.Element => {
+    const encodedQuestion = encodeURIComponent(question)
+    const maxUrl = `https://app.posthog.com/#panel=max:${encodedQuestion}`
```

Comment:
nit: this assumes the user has a US-based installation, I think. I'm not sure if we can detect which cloud the user is signed up for, from the docs.

Reply:
That's why we link to `app.posthog.com` - so we force a redirect to either US or EU. 

I'm also trying to get that redirect to work better, after a few complaints that it's easy to miss: https://github.com/PostHog/posthog/pull/30747